README.md

```diff
@@ -1,47 +1,43 @@
-To use this template create a new repository and select `example-pkg` from the template dropdown. 
-Make sure to name your repository following `<name>-pkg` convention.
-If you're interested in automated dependency updates make sure that `renovate` has access to the new repository.
-
-# Things to do change when you use this repository as a template:
-- [ ] replace `example-pkg` with the name of your package in this `README`
-- [ ] update the `@smg-automotive/example` with the name of your package in `package.json`
-- [ ] update `repository` and `homepage` sections in `package.json` to point to your GitHub repository
-- [ ] ensure that branch protection rules are applied to the `main` branch (Settings > Branches)
-  - Require a pull request before merging
-  - Require approvals
-  - Require status checks to pass before merging (project needs to be built in circle for the checks to show up)
-  - Require linear history
-  - Include administrators
-- [ ] enable `dependabot` for security updates (Settings > Code security and analysis)
-- [ ] ensure that Frontend team has the admin access to the repository (Settings > Collaborators and teams)
-- [ ] ensure that the Bots team has write access to the repository (this is needed to release the package)
-- [ ] provide usage examples in the `README.md`
-- [ ] enable the project on circleci.com to build and test your package
-- [ ] change the circleci status badge in this `README` to the new project
-- [ ] develop an awesome package
-- [ ] live long and prosper
-# example-pkg
-
-[![CircleCI](https://circleci.com/gh/smg-automotive/example-pkg/tree/main.svg?style=svg&circle-token=c183f151fea3c74453cf8dd962d31e115906a300)](https://circleci.com/gh/smg-automotive/example-pkg/tree/main)
+
+# next-page-bundlesize-pkg
+## Page-level bundle size check for next.js
+
+
+[![CircleCI](https://circleci.com/gh/smg-automotive/next-page-bundlesize-pkg/tree/main.svg?style=svg&circle-token=c183f151fea3c74453cf8dd962d31e115906a300)](https://circleci.com/gh/smg-automotive/next-page-bundlesize-pkg/tree/main)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 
 ## Usage
 ```
-npm install @smg-automotive/example
+npm install @carforyou/next-page-bundlesize -D
+next build
+npx next-page-bundlesize --maxSize="200 kB" --buildDir=.next
 ```
 
+When using [`@carforyou/configuration`](https://github.com/carforyou/carforyou-configuration-pkg/) and using a stage-specific build dir:
```

Comment:
Update to new config package, so far we don't use a specific build dir.
https://github.com/smg-automotive/configuration-pkg